content/outcomes/application-development/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Build Modern Applications
+weight: 1
+layout: single
+description: "Learn how we approach application development at VMware Tanzu Labs"
+team:
+  - VMware Tanzu Labs
+---
+ 
+## Why is this learning path important?
```

Comment:
Do we need to say learning path? Or is `Why is this important?` sufficient?

Also note, you've capitalized _Learning Path_ in the next section header but not in this one. We should be consistent.

Reply:
Good catch. Fixed.